config/lib/app.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var config = require('../config'),
+	mongoose = require('./mongoose'),
+	express = require('./express'),
+	chalk = require('chalk');
+
+// Initialize Models
+mongoose.loadModels();
```

Comment:
It's required to load the models so that they are already available for the rest of the code/tests when we do `User = mongoose.model('user')`.
I also created an exported function for it in-case some scenario will require just loading the models and not requiring to init() or start() the app.